bokehjs/src/coffee/range/data_range1d.coffee

```diff
@@ -35,8 +32,13 @@ class DataRange1d extends Range1d.Model
       '_end', 'flipped', '_auto_end', 'range_padding', 'default_span'
     ])
 
-    @register_property('auto_ranging', @_get_auto_ranging, true)
-    @add_dependencies('start', this, ['_start', '_end'])
+    if attrs?.start?
+      @set('start', attrs.start)
+      attrs.start = undefined
```

Comment:
"start" and "end" are properties defined above. I'm not exactly sure what will happen if they were left in `attrs` and passed to the `HasProperties` initializer  in `super` but I know it would not be anything desired. If there is a better spelling for "delete this attribute from this object" I'm all for changing it, but I think they need to be removed, somehow.